### DISCUSSION TITLE
Any black friday sales for individual purchases?

### DISCUSSION THREAD

**Discussion by shawngiese** [GetPublii/Publii#1826] [General] [answered]
Like a couple of plugins and a theme?

**Comment by VoxNumana** [comment]
I agree. Why on earth would I want to be a theme bundle? Just one theme will do for me. Why not have a discount for one theme? Absolutely crazy thinking by the Publii team

**Reply by michaelfluegge** [reply]
one theme costs 35€. Have you ever built one by yourself? It costs you hours and hours, so 35€ is nothing for a theme. 
What discount do you have in mind? 10% = 3.50€, 20% = 7.00€. .... What are we talking about ?
As a VoxNumana, is was not very wisely spoken, from my point of view.
But I am 100% with you, nobody needs this bundle of themes, one theme would be enough.

**Reply by VoxNumana** [reply]
Why would I want to build a theme? Publii sells themes to people who do not want to build their own. Publii does the same theme bundle discount every year. If they did it for one theme only they would probably get a much better response

**Reply by michaelfluegge** [reply]
or maybe they get such a good response they probably don’t need to discount single themes

**Reply by michaelfluegge** [reply]
And, by the way, if it is money, you want to save: I wrote about 10 lines of code to make a free theme to look like the paid one.

**Reply by shawngiese** [reply]
I was looking at 2 plugins and a theme. Now with this bundle discount I would end up spending even more to get those same things. Moved me from instant purchase to wait and decide if I really need any of them.

**Reply by michaelfluegge** [reply]
@bobmitro, @dziudek , any thoughts on this marketing issue?

**Reply by VoxNumana** [reply]
The other problem is when Publii gets updated (a year after your theme bundle purchase), you will need to pay again for EVERY theme to update to current version number.

**Comment by bobmitro** [comment, marked as answer]
@VoxNumana 
It is not true that updating Publii requires repurchasing every theme. There is no obligation to update themes unless you want to take advantage of new features introduced in Publii. Over the past two years, we’ve introduced only two major functionalities – plugins and pages – that might require theme updates. Other than that, themes can be used as-is, regardless of Publii updates.

We also ensure backward compatibility with every release. This means you can continue using your theme for as long as you like without updating it. For example, the Simple theme, released several years ago, still works perfectly with the latest version of Publii. You can see it in action here: https://demo.getpublii.eu/themes/simple/v1/.

Additionally, lots of Publii stuff is completely free – the app itself, themes, and plugins. We provide free support for users and operate on a modest revenue, just enough to sustain development.

So, it’s puzzling why you claim that themes need to be updated with every Publii release. This is simply false and misleading, as it doesn’t reflect how Publii or our themes actually work.

---

Regarding Black Friday sales, we’d like to clarify our sales model. The production cost of a single premium theme ranges between €2,000 and €4,000. So, offering 20-40% discounts is simply not viable for us. We view theme purchases more as a way to support the Publii project rather than a significant source of income. 

The revenue from premium themes doesn’t cover the full cost of maintaining Publii. Much of the development is done in our free time, as we work full-time jobs to sustain our livelihoods. 

We sincerely appreciate the support from our users, but it’s important to understand that Publii is a labor of love, not a profit-driven enterprise. Our focus remains on delivering quality tools and features while ensuring accessibility for everyone.
